### ISSUE TITLE
Welch's Method for IAAFT or AAFT

### ISSUE THREAD

**Issue by duodenum96** [JuliaDynamics/TimeseriesSurrogates.jl#162]
The neurophysiological data I'm analyzing have long time series (let's say 5-10 minutes) however the frequencies of interest tend to be between 1 to 40 Hz. A common practice in our field is to estimate the power spectrum via Welch's method of averaging periodograms. Would doing the same be worthwile in IAAFT method? If so, I would be willing to contribute this to the package, since I'm already going to implement it for my own project.

**Comment by Datseris** [comment]
Hi,

it is not clear to me what you wish to contribute :S ...

Do you want to contribute a new method for computing Power Spectra? If so, this doesn't really belong in this package. Probably DSP.jl makes more sense. 

Do you instead want to _enhance_ the current `IAFFT` algorithm to instead of use the standard FFT to compute the spectrum, to instead use the Welch's method? Sure, that can go in here if you think it is worthwhile, i.e., it increases either teh accuracy or the computational performance of the rsults.

**Comment by duodenum96** [comment]
Of course, you are right. I wrote a small matlab code that does this and it is working as intended. I am getting familiar with the coding style of TimeseriesSurrogates.jl now, I will implement it in Julia after I figure out the process.

**Comment by kahaaga** [comment]
Excellent! Looking forwards to your contribution :D

**Comment by duodenum96** [comment]
Hello again,

Just an update: I no longer need this functionality and since this is probably not very useful for the rest of the community, I think we can close this issue. Sorry for taking your time about it.
